core/src/player.rs

```diff
@@ -2083,6 +2078,14 @@ impl PlayerBuilder {
         self
     }
 
+    /// Sets the stage scale mode and optionally prevents movies from changing it.
+    #[inline]
+    pub fn with_scale(mut self, scale: StageScaleMode, force: bool) -> Self {
```

Comment:
To avoid confusion, I prefer calling the introduced fields and methods `scale_mode` and not just `scale`.

Reply:
I've renamed the fields and methods accordingly, besides the user-facing options, which I wanted to name consistently with the Flash embed option (`scale`): https://helpx.adobe.com/flash/kb/flash-object-embed-tag-attributes.html